### PR TITLE
updated the insert_editor_css to insert_global_admin_css

### PR DESCRIPTION
**Pull request by blazete**
Fixes #...

_Please check the following:_

-   [x] Do the tests still pass?[^1]
-   [x] Does the code comply with the style guide?
    -   [ ] Run `make lint` from the Wagtail root.
-   [x] For Python changes: Have you added tests to cover the new/fixed behaviour?
-   [x] For front-end changes: Did you test on all of Wagtail’s supported environments?[^2]
    -   [x] **Please list the exact browser and operating system versions you tested**:
    -   [x] **Please list which assistive technologies [^3] you tested**:
-   [x] For new features: Has the documentation been updated accordingly?

**Please describe additional details for testing this change**.

[^1]: [Development Testing](https://docs.wagtail.org/en/latest/contributing/developing.html#testing)
[^2]: [Browser and device support](https://docs.wagtail.org/en/latest/contributing/developing.html#browser-and-device-support)
[^3]: [Accessibility Target](https://docs.wagtail.org/en/latest/contributing/developing.html#accessibility-targets)

### CODE REVIEW

**Comment by blazete** [comment]
Done with the issue

**Comment by Scotchester** [comment]
Which issue?

**Comment by lb-** [comment]
@blazete  it would be good to understand the reasoning behind this change? What problem did you run into that lead you to do this?

Please have a read about how best to work towards your first contribution, it has some helpful tips for how to prepare a PR.

https://docs.wagtail.org/en/latest/contributing/first_contribution_guide.html

**Comment by thibaudcolas** [comment]
I believe this is for #10402. Thank you for giving this a go @blazete.

File: wagtail/admin/templates/wagtailadmin/userbar/base.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
     <div class="w-userbar w-userbar--{{ position|default:'bottom-right' }}" data-wagtail-userbar part="userbar">
         <link rel="stylesheet" href="{% versioned_static 'wagtailadmin/css/core.css' %}">
         {% hook_output 'insert_global_admin_css' %}
-        {% hook_output 'insert_editor_css' %}
+        {% hook_output 'insert_global_admin_css' %}
```
Comment (RealOrangeOne) [inline wagtail/admin/templates/wagtailadmin/userbar/base.html:7]: Issue: Duplicated hook outputs

File: wagtail/admin/templates/wagtailadmin/admin_base.html [hunk 2]
```diff
@@ -5,7 +5,7 @@
     <link rel="stylesheet" href="{% versioned_static 'wagtailadmin/css/core.css' %}">
 
     {% hook_output 'insert_global_admin_css' %}
-    {% hook_output 'insert_editor_css' %}
+    {% hook_output 'insert_global_admin_css' %}
```
Comment (RealOrangeOne) [inline wagtail/admin/templates/wagtailadmin/admin_base.html:8]: Issue: Duplicated hook outputs

File: docs/reference/hooks.md [hunk 3]
```diff
@@ -434,9 +434,9 @@ Hooks for customising the editing interface for pages and snippets.
 
 Rich text fields in Wagtail work with a list of 'feature' identifiers that determine which editing controls are available in the editor, and which elements are allowed in the output; for example, a rich text field defined as `RichTextField(features=['h2', 'h3', 'bold', 'italic', 'link'])` would allow headings, bold / italic formatting and links, but not (for example) bullet lists or images. The `register_rich_text_features` hook allows new feature identifiers to be defined - see [](rich_text_features) for details.
 
-(insert_editor_css)=
+(insert_global_admin_css)=
 
-### `insert_editor_css`
+### `insert_global_admin_css`
```
Comment (RealOrangeOne) [inline docs/reference/hooks.md:439]: Issue: `insert_global_admin_css` is already documented, there's no need to rename the existing one.

https://docs.wagtail.org/en/stable/reference/hooks.html#id21

**Comment by gasman** [comment]
Will close this, as it isn't the correct approach for fixing #10402 - it will cause any uses of `insert_editor_css` on existing projects to stop working immediately (it needs to go through a deprecation period before being dropped fully) and any uses of `insert_global_admin_css` will be output twice, as @RealOrangeOne noted. An updated fix is now in progress at #10499.

**Pull request closed without merging**